### DISCUSSION TITLE
Vue Setup

### DISCUSSION THREAD

**Discussion by xeno** [hannoeru/vite-plugin-pages#305] [Q&A] [answered]
[Here's my repository](https://github.com/SomewhereBeyond/survey)

I feel like I'm missing a step. I've set up my application with Vite + Vue, but App.vue is still taking precedent. Am I supposed to be adding something to `main.js`?

Thanks!

**Comment by marekvospel** [comment, marked as answer]
Yes, you need to add `<RouterView />` where you want your routes to be mounted.
